mwc-random.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version:  3.0
 build-type:     Simple
 name:           mwc-random
-version:        0.15.0.2
```

Comment:
That would be minor bump to 0.15.1.0 not patch bump. We're adding new function

Reply:
Done